src/components/Account/Templates/TemplateItem.vue

```diff
@@ -52,7 +65,11 @@ export default {
         deleteTemplate () {
             Util.snackbar("Deleted template: " + this.text);
```

Comment:
```suggestion
            Util.snackbar('Deleted template: ' + this.text);
```

Also, prefer template literal to string concat